packages/lablgtk3/lablgtk3.3.0.beta4/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+
+description: "OCaml interface to GTK+"
+synopsis: "OCaml interface to GTK+"
+
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "http://lablgtk.forge.ocamlcore.org/"
+bug-reports: "https://github.com/garrigue/lablgtk/issues"
+dev-repo: "git+https://github.com/garrigue/lablgtk.git"
+license: "LGPL with linking exception"
+doc: "https://garrigue.github.io/lablgtk/lablgtk"
+
+depends: [
+  "ocaml"     {         >= "4.05.0" }
+  "dune"      { build & >= "1.4.0"  }
+  "conf-gtk3" { build & >= "18"     }
```

Comment:
Is really con-gtk3 a build dep? I may have been the one that added this, but I am not sure it is correct.

Reply:
I think this is correct. It checks that gtk3 is available, and installs it if necessary. Of course the library itself is also necessary at runtime, but if you have installed it it will still be around. Or are there binary packages for opam?

Reply:
Indeed I think you are right, if `conf-gtk3` is updated, `lablgtk3` shouldn't be rebuilt. But indeed there is no way in opam to specify a true system dep.